### PR TITLE
Feature/802 generate apk

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: .github/workflows/build-apk.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  push:
+    branches:
+      - feature/802-generate-apk
+
+jobs:
+  build-android:
+    runs-on: ubuntu-20.04
+    env:
+      NODE_ENV: demo
+      EXPO_TOKEN: ${{ secrets.EXPO_TOKEN }}
```
Comment (theblockstalk) [inline .github/workflows/build-apk.yaml:13]: remove NODE_ENV it is not used
use EXPO_NODE_ENV and also EXPO_PLATFORM env which are both used for build

Comment (theblockstalk) [reply .github/workflows/build-apk.yaml:13]: also do a rebase to get fix https://github.com/Tonomy-Foundation/Tonomy-ID/pull/806

File: .github/workflows/build-apk.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: 🚀 Build and publish tonomy app
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.1
+      - run: npm ci
+      - run: npm run build:prepare
+      - run: npx eas build --clear-cache --profile demo --platform android --local
```
Comment (theblockstalk) [inline .github/workflows/build-apk.yaml:26]: I don't think we can increase runner size
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

File: .github/workflows/build-apk.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      - name: 🚀 Build and publish tonomy app
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.1
+      - run: npm ci
+      - run: npm run build:prepare
+      - run: npx eas build --clear-cache --profile development --platform android --local
+
+      - name: Find APK files
+        run: find . -name "*.apk" > apk_files.txt
```
Comment (theblockstalk) [inline .github/workflows/build-apk.yaml:29]: `find . -name "*.apk" -exec mv {} TonomyID.apk \;`
remove L31-37

File: prepare.app.config.ts [hunk 4]
```diff
@@ -76,7 +76,7 @@ const expo: ExpoConfig = {
     ],
     extra: {
         eas: {
-            projectId: '48982c5e-41ff-4858-80ea-8e1b64092f21',
```
Comment (theblockstalk) [inline prepare.app.config.ts:79]: change this back to previous

Comment (sadiabbasi) [reply prepare.app.config.ts:79]: For the TonomyDev expo account i created development slug and used id from the tonomy-id-dev

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**